src/app/app.config.ts

```diff
@@ -1,15 +1,19 @@
-import { ApplicationConfig, provideZoneChangeDetection } from '@angular/core';
+import { ApplicationConfig, provideAppInitializer, provideZoneChangeDetection } from '@angular/core';
 import { provideRouter } from '@angular/router';
 
 import { routes } from './app.routes';
 import { provideAnimationsAsync } from '@angular/platform-browser/animations/async';
 import { provideHttpClient } from '@angular/common/http';
+import { initializeMaterialSymbols } from './core/initializers/initialize-material-symbols';
+import { initializeSvgIcons } from './core/initializers/initialize-svg-icons';
 
 export const appConfig: ApplicationConfig = {
     providers: [
         provideZoneChangeDetection({ eventCoalescing: true }),
         provideRouter(routes),
         provideAnimationsAsync(),
         provideHttpClient(),
+        provideAppInitializer(() => initializeMaterialSymbols()),
```

Comment:
These initializer functions are cool, I could see myself do this for some functionality I kind of have spread out across a couple "root" components in projects